src/admin/messages/settings.html

```diff
@@ -0,0 +1,49 @@
+<motech-list controller="MessagesController">
+    <motech-list-header>
+        <h5 class="text-plain inside">{{'admin.messages.notifications.header' | translate}}</h5>
+    </motech-list-header>
+    <motech-list-item ng-repeat="rule in notificationRuleDto.notificationRules">
+        <fieldset class="form-inline">
+            <div class="form-group"> {{'admin.messages.notifications.notificationType' | translate}}
+                <select ng-model="rule.actionType">
+                    <option ng-repeat="available in ['SMS', 'EMAIL']" ng-click="changeRuleActionType(rule, available)" value="{{available}}">
+                        {{'admin.messages.notifications.' + available | translate}}
+                    </option>
+                </select>
+            </div>
+            <div class="form-group"> {{'admin.messages.notifications.level' | translate}}
+                <select ng-model="rule.level">
+                    <option ng-repeat="available in ['CRITICAL', 'ERROR', 'WARN', 'INFO', 'DEBUG']" ng-click="changeRuleLevel(rule, available)" value="{{available}}">
+                        {{'admin.messages.' + available | translate}}
+                    </option>
+                </select>
+            </div>
+            <div class="form-group"> {{'admin.messages.moduleName' | translate}}
+                <select ng-model="rule.moduleName">
+                    <option>
+                        {{'admin.messages.all' | translate}}
+                    </option>
+                    <option ng-repeat="bundle in bundles" ng-click="changeRuleModuleName(rule, bundle)" value="{{bundle.name}}">
+                        {{bundle.name}}
+                    </option>
+                </select>
+            </div>
+            <div class="form-group" ng-class="{'has-error': !rule.recipient}">
+                <span class="control-label offset-1">{{'admin.messages.notifications.recipient' | translate}}</span>
+                <input class="form-control input-auto input-large-fluid" ng-model="rule.recipient" type="text" value="" />
+            </div>
+            <div class="form-group">
+            <a ng-click="removeRule(rule)" class="btn btn-danger"><i class="fa fa-trash-o"></i></a>
+            </div>
+        </fieldset>
+    </motech-list-item>
+    <motech-list-footer>
+        <div class="btn-group">
+            <button class="btn btn-success" ng-click="newRule()"><i class="fa fa-plus"></i>&nbsp;{{'admin.messages.notifications.newRecipient' | translate}}</button>
```

Comment:
Here something like this would work to simplify the code using classes from common files
`<button class="button-add" ng-click="newRule()">{{'admin.messages.notifications.newRecipient' | translate}}</button>`